bindata/network/ovn-kubernetes/004-config.yaml

```diff
@@ -1,11 +1,26 @@
 ---
-# The network cidr and service cidr are set in the ovn-config configmap
+# The config file used by ovnkube is passed as a configmap.
 kind: ConfigMap
 apiVersion: v1
 metadata:
-  name: ovn-config
+  name: ovnkube-config
   namespace: openshift-ovn-kubernetes
 data:
-  net_cidr:   {{.OVN_cidr}}
-  svc_cidr:   {{.OVN_service_cidr}}
-  k8s_apiserver: "{{.K8S_APISERVER}}"
+  ovn_k8s.conf:   |-
+    [default]
+    mtu="{{.MTU}}"
+    cluster-subnets="{{.ClusterSubnets}}"
+
+    [kubernetes]
+    service-cidr="{{.ServiceCidr}}"
+    ovn-config-namespace="openshift-ovn-kubernetes"
+    apiserver="{{.K8S_APISERVER}}"
+
+    [logging]
+    loglevel="{{.OvnkubeLogLevel}}"
```

Comment:
This should be overridable via the environment config map trick. So we need to pass loglevel via environment args.

Reply:
OK.

Reply:
Should this still be here?

Reply:
Yeah, I think you should delete this, since we pass `--loglevel` to both processes.